src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -131,7 +131,7 @@ public void trackVisionTape() {
     }
 
     public void teleopPeriodic() {
-        arcadeDrive(ButtonMap.getThrottle(), ButtonMap.getTurn());
```

Comment:
We need to look at exactly what is causing this on different controllers but we probably shouldn't check-in this band-aid on this commit